pallets/weights/src/pallet_asset.rs

```diff
@@ -111,12 +111,12 @@ impl pallet_asset::WeightInfo for WeightInfo {
     // Storage: Asset Identifiers (r:0 w:1)
     // Storage: ExternalAgents AgentOf (r:0 w:1)
     // Storage: ExternalAgents GroupOfAgent (r:0 w:1)
-    fn create_asset(_n: u32, i: u32, f: u32) -> Weight {
-        (207_235_000 as Weight)
-            // Standard Error: 10_000
-            .saturating_add((66_000 as Weight).saturating_mul(i as Weight))
-            // Standard Error: 41_000
-            .saturating_add((100_000 as Weight).saturating_mul(f as Weight))
+    fn create_asset(_n: u32, i: u32, f: u32, ) -> Weight {
```

Comment:
The `n` parameter is the asset name length.  The tx per-byte fee should cover this.